tests/actions/labOrderTest/labConceptsAction.test.js

```diff
@@ -0,0 +1,99 @@
+import {
+  fetchLabConceptsSuccess,
+  fetchLabConceptsFailure,
+  fetchLabConcepts
+} from '../../../app/js/actions/labOrders/labConceptsAction';
+import {
+  FETCH_LAB_CONCEPTS_SUCCESS,
+  FETCH_LAB_CONCEPTS_FAILURE,
+  FETCH_LAB_CONCEPTS_LOADING,
+} from '../../../app/js/actions/actionTypes';
+
+describe('Getting Lab Concepts action', () => {
+  it('should create an action to log failure', () => {
+    const error = 'No network connection';
```

Comment:
please make your test-case messages more descriptive, mentioning the actual action that would be created would make this test-case message more explanatory

Reply:
Thanks, @Efosaok.
I would update the test description to reflect the actual action that is being dispatched.